packages/ckeditor5-dev-build-tools/tests/plugins/splitCss/fixtures/nested-css-variables/file1.css

```diff
@@ -0,0 +1,12 @@
+:root {
+	--ck-spacing-unit: var(--ck-variable-1);
+	--ck-variable-1: var(--ck-variable-2);
+	--ck-variable-2: var(--ck-variable-3);
+	--ck-variable-3: var(--ck-variable-4);
+	--ck-variable-4: var(--ck-variable-5);
+	--ck-variable-5: 0.6em;
+}
```

Comment:
I would add a few more assertions:

1. `--ck-variable-x` that uses `--ck-variable-y` but both aren't used anywhere.
2. A variable that is a default value if another variable does not exist: `var( --ck-variable-a, var( --ck-variable-b ) )`.

Reply:
Done :heavy_check_mark: